tgui/packages/tgui/interfaces/Who.jsx

```diff
@@ -142,48 +124,83 @@ const GetAddInfo = (props, context) => {
   );
 };
 
+const FilterPlayers = (props, context) => {
+  const { players_to_filter } = props;
+
+  return players_to_filter.map((clientObj) => {
+    const ckey = Object.keys(clientObj)[0];
+    return <GetPlayerInfo key={ckey} ckey={ckey} {...clientObj[ckey][0]} />;
+  });
+};
+
 const GetPlayerInfo = (props, context) => {
-  const { act } = useBackend(context);
-  const {
-    admin,
-    player: { ckey, ckey_color, color, text },
-  } = props;
-  return admin !== 0 ? (
+  const { ckey, text, color, ckey_color } = props;
+
+  return (
     <Button
-      color={'transparent'}
+      color="transparent"
       style={{
-        'border-color': admin ? color : '#2185d0',
-        'border-style': 'solid',
-        'border-width': '1px',
-        color: admin ? color : ckey_color,
+        borderColor: color || '#2185d0',
+        borderStyle: 'solid',
+        borderWidth: '1px',
+        color: color || ckey_color || '#2185d0',
       }}
       onClick={() => act('get_player_panel', { ckey: ckey })}
```

Comment:
Clicking on any player button throws 
![image](https://github.com/user-attachments/assets/6b45270f-d32f-489b-beb2-eedb116dbd56)
![image](https://github.com/user-attachments/assets/6ca5b8f2-4b8c-4729-a0d1-3c76f8aca9bf)
because `useBackend` is absent from this function.